app/src/main/java/com/example/android/unsplash/ui/grid/PhotoAdapter.java

```diff
@@ -84,11 +84,17 @@ public void onClick(View v) {
                 View navBackground = decorView.findViewById(android.R.id.navigationBarBackground);
                 Pair statusPair = Pair.create(statusBackground,
                         statusBackground.getTransitionName());
-                Pair navPair = Pair.create(navBackground, navBackground.getTransitionName());
-
-                host.startActivity(intent,
-                        ActivityOptions.makeSceneTransitionAnimation(host,
-                                authorPair, photoPair, statusPair, navPair).toBundle());
+                ActivityOptions options;
+                // some phone doesn't has a 'navigation bar' so we do some extra check
```

Comment:
Please change to:
"Some devices don't have a navigation bar, so we have to check if it's available."